### DISCUSSION TITLE
Problem with MinifyAll Extension in VSCodium

### DISCUSSION THREAD

**Discussion by sapereaudete** [VSCodium/vscodium#1998] [General]
What would be the correct way to announce a bug? I opened a discussion [here](https://github.com/Josee9988/MinifyAll/discussions/186), but was the way it should be?

**Comment by bvwidt** [comment]
@sapereaudete Use a [Bug Report](https://github.com/VSCodium/vscodium/issues/new?assignees=&labels=bug&projects=&template=bug_report.md&title=) 😉

**Comment by daiyam** [comment]
No, it's an issue with the extension.
